src/Compilers/CSharp/Portable/Symbols/MergedNamespaceSymbol.cs

```diff
@@ -168,17 +168,42 @@ private ImmutableArray<Symbol> SlowGetChildrenOfName(ReadOnlyMemory<char> name)
         /// </summary>
         private HashSet<ReadOnlyMemory<char>> SlowGetChildNames(IEqualityComparer<ReadOnlyMemory<char>> comparer)
         {
-            var childNames = new HashSet<ReadOnlyMemory<char>>(comparer);
+            // compute the final capacity of the set we'll return, to reduce heap churn
+            int childCount = 0;
+            List<ImmutableArray<Symbol>> namespacesMembers = new(capacity: _namespacesToMerge.Length);
```

Comment:
>new [](http://example.com/codeflow?start=61&length=3)

We are not allowing target typed `new` under Compilers.

Reply:
Good to know